Framework/Utils/test/test_RootTreeWriter.cxx

```diff
@@ -207,12 +207,12 @@ BOOST_AUTO_TEST_CASE(test_RootTreeWriter)
   std::vector<InputRoute> schema = {
     {InputSpec{"input1", "TST", "INT"}, 0, "input1", 0},       //
     {InputSpec{"input2", "TST", "CONTAINER"}, 1, "input2", 0}, //
-    {InputSpec{"input3", "TST", "CONTAINER"}, 2, "input3", 1}, //
+    {InputSpec{"input3", "TST", "CONTAINER"}, 2, "input3", 0}, //
```

Comment:
Previously, a different time slice id had to be used because DPL DataRelayer did not support multiple messages per route and time slice. The intention of this check is to make sure that two branches are created from the two messages on this route. So its fine to put the two messages in the same time slice now.